2025-01-29 15:06:56,245 - kpg_malibu_bvb - ERROR - Error starting bot: 'NoneType' object has no attribute 'run_daily'
Traceback (most recent call last):
  File "/Users/alexey/Documents/Python/Актуальное/KPG_Malibu_BVB_bot/main.py", line 113, in run
    job_queue.run_daily(
    ^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'run_daily'
2025-01-29 15:10:47,820 - kpg_malibu_bvb - INFO - Bot started
2025-01-29 15:13:03,005 - kpg_malibu_bvb - INFO - Bot started
2025-01-29 15:19:20,705 - kpg_malibu_bvb - INFO - Command: help | User: 43704386 | Chat: 43704386
2025-01-29 15:25:53,202 - kpg_malibu_bvb - INFO - Command: create_session | User: 43704386 | Chat: -1001931349972
